From: Alexander Larsson <alexl@redhat.com>
Date: Thu, 19 Nov 2020 17:22:07 +0100
Subject: transaction: Add more details to the ABORTED error

This is reported when we reported an error to the user via
::operation-error signal and the app told us to not continue.

If this happens in some weird case and we see the results its nice
to have access to the original error message.

Origin: upstream, 1.9.2, commit:09b470c0fd8a5392173b16009c90c51e4582cb4b
---
 common/flatpak-transaction.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/flatpak-transaction.c b/common/flatpak-transaction.c
index bcfb4e7..26b44bd 100644
--- a/common/flatpak-transaction.c
+++ b/common/flatpak-transaction.c
@@ -4791,7 +4791,7 @@ flatpak_transaction_real_run (FlatpakTransaction *self,
                   break;
                 }
 
-              flatpak_fail_error (error, FLATPAK_ERROR_ABORTED, _("Aborted due to failure"));
+              flatpak_fail_error (error, FLATPAK_ERROR_ABORTED, _("Aborted due to failure (%s)"), local_error->message);
               succeeded = FALSE;
               break;
             }
